redirects.js

```diff
@@ -2,9 +2,14 @@
  * Redirects in this file are intended to be for documentation content only. The redirects will be applied to developer.hashicorp.com.
  */
 module.exports = [
-  // {
-  //   source: '/terraform/docs/path',
-  //   destination: '/terraform/docs/new/path',
-  //   permanent: true,
-  // },
+   {
+     source: '/terraform/enterprise/policy-enforcement/:path*',
+     destination: '/terraform/enterprise/sentinel/:path*',
+     permanent: true,
+   },
+  {
+     source: '/terraform/enterprise/policy-enforcement/sentinel/:path*',
+     destination: '/terraform/enterprise/sentinel/:path*',
+     permanent: true,
+   }
```

Comment:
I think these should be _temporary_

```suggestion
   {
     source: '/terraform/enterprise/policy-enforcement/:path*',
     destination: '/terraform/enterprise/sentinel/:path*',
     permanent: false,
   },
  {
     source: '/terraform/enterprise/policy-enforcement/sentinel/:path*',
     destination: '/terraform/enterprise/sentinel/:path*',
     permanent: false,
   }
```